### DISCUSSION TITLE
subset data cube from list of specific lat and lon values

### DISCUSSION THREAD

**Discussion by gkinglisboa** [SciTools/iris#4869] [Q&A] [answered]
I am a newbie.  My experience is matlab.  
In matlab the following is easy to do, but I cannot see how to do it with Iris (or any other netcdf tool).  

I have a data cube of sea surface temperatures sst(lat, lon, time) in a region of the coastal ocean,  
and a list of specific  lat, lon values (along a curvy line close to the coast). 
I want to extract the SST at these lats and lons from the data cube for each time value.

How do I extract this subset of data using Iris??

Hope you can help.

**Comment by trexfeathers** [comment]
Hi @gkinglisboa, have you looked at the [`trajectory`](https://scitools-iris.readthedocs.io/en/latest/generated/api/iris/analysis/trajectory.html) module? Are the points you describe exact matches, or are you looking for the nearest points to your points? Thanks!

**Comment by gkinglisboa** [comment]
Thanks Martin, trajectory module looks interesting. Will investigate.

greg

**Comment by gkinglisboa** [comment, marked as answer]
I cannot find any examples of using the trajectory module, waypoints, etc.
Do you know where I can find some?

**Reply by trexfeathers** [reply]
The only examples I am aware of are the small ones in the webpage I linked originally. I would hope these were enough to help 'have a go' at your use case, if only to see what output you get. Is there something you need specifically?

**Reply by gkinglisboa** [reply]
Hi Martin, I eventually found an example that was close to my case.  I will have to work through it and expand on the comments on the example to see how it works, but it does look promising.  

https://ocefpaf.github.io/python4oceanographers/blog/2015/04/06/bathymetry/

**Comment by pp-mo** [comment]
Archived, since answered
